src/vmm/src/cpuid/brand_string.rs

```diff
@@ -4,7 +4,7 @@
 use std::arch::x86_64::__cpuid as host_cpuid;
 use std::slice;
 
-use crate::common::{VENDOR_ID_AMD, VENDOR_ID_INTEL};
+use crate::cpuid::common::{VENDOR_ID_AMD, VENDOR_ID_INTEL};
```

Comment:
Maybe it is worth converting all `crate::cpuid` into just `super`? That way all inner module paths are self contained. And if we ever need to move this module anywhere else, it would be much easier.

Reply:
Some can be changed to `super` but most end up looking like `super::super::super` which I don't think is ideal.

Reply:
Well.. we already have `super::super` somewhere as I recall. But I was thinking about always using the shortest path. So if module need access to itself it uses `super::something` instead of `crate::module::something`. Maybe it is worth having some verbal rule about it.